### PR TITLE
Fix links to PowerShell on install docs

### PR DESCRIPTION
**Pull request by jokajak**
## PR Summary
Update Install doc links.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by jokajak** [comment]
I failed to use the present tense and imperative mood when generating the change in my browser. Is that cause to reject this small PR?

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ At the end of this exercise, you should be able to launch the PowerShell session
 
 [inst-linux]: ../installation/linux.md
 [inst-win]: ../installation/windows.md
-[inst-macos]: ../installation/macos.md
+[inst-macos]: https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md
```
Comment (daxian-dbw) [inline docs/learning-powershell/README.md]: Please use `https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos?view=powershell-6` instead.

Comment (bergmeister) [reply docs/learning-powershell/README.md]: Please use `https://docs.microsoft.com/powershell/scripting/setup/installing-powershell-core-on-macos?view=powershell-6` (i.e. without the `en-us` culture) instead.

File: docs/learning-powershell/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ At the end of this exercise, you should be able to launch the PowerShell session
 
 [inst-linux]: ../installation/linux.md
 [inst-win]: ../installation/windows.md
```
[the remark this answers is not in the document]
Comment (bergmeister) [reply docs/learning-powershell/README.md]: The links are:
- Windows: https://docs.microsoft.com/powershell/scripting/setup/installing-powershell-core-on-windows
- Linux: https://docs.microsoft.com/powershell/scripting/setup/installing-powershell-core-on-linux

**Review by bergmeister — APPROVED** [review]

**Review by iSazonov — APPROVED** [review]

**Pull request merged**